public/gameInfo.php

```diff
@@ -56,6 +57,56 @@
 
 $gameAlts = getGameAlternatives($gameID);
 
+$v = requestInputSanitized('v', 0, 'integer');
+if ($v != 1) {
+    foreach ($gameAlts as &$gameAlt) {
+        if ($gameAlt['Title'] == '[Theme - Mature]') {
+            if (getAccountDetails($user, $accountDetails) &&
+          BitSet($accountDetails['websitePrefs'], UserPref::SiteMsgOff_MatureContent)) {
+                break;
+            }
+
+            RenderHtmlStart(true); ?>
+<head prefix="og: http://ogp.me/ns# retroachievements: http://ogp.me/ns/apps/retroachievements#">
+    <?php RenderSharedHeader(); ?>
+    <?php RenderTitleTag($pageTitle); ?>
+    <?php RenderGoogleTracking(); ?>
+</head>
+<body>
+<?php RenderTitleBar($user, $points, $truePoints, $unreadMessageCount, $errorCode, $permissions); ?>
+<?php RenderToolbar($user, $permissions);
+            echo "<div id='mainpage'>";
+            echo "<div id='leftcontainer'>";
```

Comment:
This should probably be `fullcontainer` so it takes up the full width of the page rather than just the left portion.

Reply:
I think it flows better as just a left container as clicking Yes will appear to flow into the left container of the actual game page and the rest of the elements will appear around the warning. But it does sort of feel like a waste of space.

Reply:
let's have it as is and iterate on that later - we should release this feature.

Reply:
Sounds good to me.